packages/react-core/src/components/Slider/Slider.tsx

```diff
@@ -316,10 +316,11 @@ export const Slider: React.FunctionComponent<SliderProps> = ({
           aria-valuenow={value}
           aria-valuetext={findAriaTextValue()}
           aria-label={thumbAriaLabel}
-          onMouseDown={handleMouseDown}
-          onTouchStart={handleToucheStart}
-          onKeyDown={handleThumbKeys}
-          onClick={onThumbClick}
+          aria-disabled={isDisabled}
```

Comment:
Also need to modify the `tabIndex` above to `-1` (or remove it) when it's disabled so it won't take keyboard focus.

Reply:
Thanks for catching this! Updated with fix.